src/NuGet.Clients/NuGet.PackageManagement.UI/UserInterfaceService/INuGetUI.cs

```diff
@@ -80,9 +79,14 @@ public interface INuGetUI : IDisposable
         bool DisplayDeprecatedFrameworkWindow { get; }
 
         /// <summary>
-        /// Package currently selected in the UI
+        /// Packages currently selected in the UI in the packages list
```

Comment:
You can't select multiple packages in this list, today. I think we don't really use this property in particular, so it may be fine to change its purpose to be "packages selected for update" - I think that's what you mean?

If so:
```suggestion
        /// Packages currently selected for update in the UI in the packages list.
```

Reply:
The idea here is to use this field in all situations, even if it is just one package for install/uninstall.

Reply:
Sure, but the comment isn't clear, and you've changed the previous meaning of this property.